jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/RangeSlidersTests.java

```diff
@@ -21,57 +23,82 @@ public void before() {
         rangeSlidersPage.checkOpened();
     }
 
-    @Test (enabled = false)
+    @Test (enabled = false, description = "Check disabled ranger slider")
     public void disabledSliderTest() {
+        boolean isVertical = disabledRangeSlider.isVertical();
         disabledRangeSlider.is().disabled();
         disabledRangeSlider.is().value(30, 60);
-        disabledRangeSlider.slideHorizontalTo(20, 40);
+        disabledRangeSlider.setValue(20, 40, isVertical);
```

Comment:
Maybe we should check inside the method if the slider is vertical instead of passing "isVertical" parameter.